### PR TITLE
OSDOCS#10240: Restructuring the Managing hosted control planes section

### PR DESCRIPTION
**Pull request by xenolinux**
This PR moves sections present under the "Managing hosted control planes" topic to relevant newly created topics.

For example, the "Configuring metrics sets for hosted control planes" section (hosted-control-planes-metrics-sets.adoc) is moved to its relevant topic "Hosted control planes Observability" (assembly file: hosted_control_planes/hcp-observability.adoc)
**NOTE:**  Content is not changed, only moved (restructured) to its topics/assemblies for better findability and relevance. 

Version(s): 4.14+

Issue: https://issues.redhat.com/browse/OSDOCS-10240

Link to docs preview: 

Changes under hosted control planes book: https://74581--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-updating

https://74581--ocpdocs-pr.netlify.app/openshift-enterprise/latest/updating/updating_a_cluster/updating-hosted-control-planes.html

QE review:
- [x] QE has approved this change.

PM review:
- [x] PM has approved this change.

Additional information:

### CODE REVIEW

**Comment by zanetworker** [comment]
lgtm

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -2316,6 +2316,10 @@ Topics:
   File: hcp-managing
 - Name: Using feature gates in a hosted cluster
   File: hcp-using-feature-gates
+- Name: Updating hosted control planes
+  File: hcp-updating
+- Name: Hosted control planes Observability
```
Comment (kcarmichael08) [inline _topic_maps/_topic_map.yml:2321]: Should "observability" be capitalized here? It's not capitalized in the module heading.

**Review by kcarmichael08 — COMMENTED** [review]
One question; otherwise lgtm!

**Pull request merged**